### PR TITLE
Shiru/log of actions

### PR DESCRIPTION
**Pull request by sylfort**
-Added log of actions to the dom
-Added animation and auto scroll to log of actions
-Added github link to footer

### CODE REVIEW

File: src/components/component/catComponent.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ import cheese4 from "@/assets/cheese_4.png";
 const catImages = [warriorCat, gangsterCat, pirateCat, wizardCat];
 const cheeseImages = [cheese1, cheese2, cheese3, cheese4];
 
-const CatComponent = ({ roundNumber }) => {
+const CatComponent = ({ addLog, clearLog }) => {
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx]: Add roundNumber to the prop

File: src/components/component/catComponent.jsx [hunk 2]
```diff
@@ -43,14 +43,13 @@ const CatComponent = ({ roundNumber }) => {
     setPlayers({});
     setCheeseAmounts({});
     setRounds(1);
-    roundNumber(0);
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx:46]: add roundNumber(0)

File: src/components/component/catComponent.jsx [hunk 3]
```diff
@@ -43,14 +43,13 @@ const CatComponent = ({ roundNumber }) => {
...
     setWinner(null);
     setScores({});
+    clearLog();
   };
 
   const handleStartGame = () => {
     setStartGame(false);
-    roundNumber(1);
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx]: add roundNumber(1)

File: src/components/component/catComponent.jsx [hunk 4]
```diff
@@ -87,7 +86,7 @@ const CatComponent = ({ roundNumber }) => {
 
   const handlePlayerSelection = (id) => {
     // Show winner after 8 rounds
-    if (rounds >= 8) {
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx]: it should be rounds >= 8

File: src/components/component/catComponent.jsx [hunk 5]
```diff
@@ -110,7 +109,6 @@ const CatComponent = ({ roundNumber }) => {
 
     // increment the number of rounds
     setRounds(rounds + 1);
-    roundNumber(rounds + 1);
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx]: Add roundNumber(rounds +1)

File: src/components/component/catComponent.jsx [hunk 6]
```diff
@@ -188,9 +186,9 @@ const CatComponent = ({ roundNumber }) => {
     console.log("Game restarted");
     handleStartGame();
     setRounds(1);
-    roundNumber(1);
```
Comment (iamzaidmohammed) [inline src/components/component/catComponent.jsx:191]: Add roundNumber(1)

**Review by iamzaidmohammed — COMMENTED** [review]
I'm done reviewing

**Review by dominic-source — APPROVED** [review]
Nice work, keep it up.

**Pull request merged**